MicroProfile-Metrics/pom.xml

```diff
@@ -55,8 +55,8 @@
 
     <properties>
         <!-- Metrics Dependencies -->
-        <microprofile.metrics.version>5.0.0</microprofile.metrics.version>
-        <microprofile.metrics.tck.version>5.0.0</microprofile.metrics.tck.version>
+        <microprofile.metrics.version>5.1.0-RC1.payara-p1</microprofile.metrics.version>
+        <microprofile.metrics.tck.version>5.1.0-RC1.payara-p1</microprofile.metrics.tck.version>
```

Comment:
5.1.0 (final) is available now

Reply:
Final will be done here: https://payara.atlassian.net/browse/FISH-7776